src/lib/common/homestore_config.fbs

```diff
@@ -250,6 +250,9 @@ table Consensus {
 
     // Timeout for data to be received after raft entry after which raft entry is rejected.
     data_receive_timeout_ms: uint64 = 10000;
+
+    // Frequency to flush durable commit LSN in millis
+    flush_durable_commit_interval_ms: uint64 = 100;
```

Comment:
I think 100 millis is too frequent, would suggest min 500 millis or 1 second.

Reply:
changed it to 500ms